client/src/views/DeviceJoinOrganizationModal.vue

```diff
@@ -2,24 +2,192 @@
 
 <template>
   <ion-page>
+    <wizard-stepper
+      v-show="pageStep > DeviceJoinOrganizationStep.Information && pageStep < DeviceJoinOrganizationStep.Finish"
+      :current-index="pageStep - 1"
+      :titles="[
+        $t('ClaimDeviceModal.stepper.GetHostCode'),
+        $t('ClaimDeviceModal.stepper.ProvideGuestCode'),
+        $t('ClaimDeviceModal.stepper.password'),
+      ]"
+    />
+    <ion-buttons
+      slot="end"
+      class="closeBtn-container"
+    >
+      <ion-button
+        slot="icon-only"
+        @click="cancelModal()"
+        class="closeBtn"
+      >
+        <ion-icon
+          :icon="close"
+          size="large"
+          class="closeBtn__icon"
+        />
+      </ion-button>
+    </ion-buttons>
+    <div
+      class="modal"
+      :class="{wizardTrue: pageStep > DeviceJoinOrganizationStep.Information && pageStep != DeviceJoinOrganizationStep.Finish}"
+    >
+      <ion-header class="modal-header">
+        <ion-title
+          v-if="titles.get(pageStep)?.title !== ''"
+          class="modal-header__title title-h2"
+        >
+          {{ titles.get(pageStep)?.title }}
+        </ion-title>
+        <ion-text
+          v-if="titles.get(pageStep)?.subtitle !== ''"
+          class="modal-header__text body"
+        >
+          {{ titles.get(pageStep)?.subtitle }}
+        </ion-text>
+      </ion-header>
+      <!-- modal content: create component for each part-->
+      <div class="modal-content inner-content">
+        <!-- part 0 (manage by JoinByLink component)-->
+        <!-- part 1 (information message join)-->
+        <div
+          v-show="pageStep === DeviceJoinOrganizationStep.Information"
+          class="step orga-name"
+        >
+          <information-join-device
+            ref="informationPage"
```

Comment:
You can remove all the `ref=""` for all pages (except for the password) since they are not referencing any variable now.

Reply:
Done